includes/hub/class-node.php

```diff
@@ -102,11 +106,12 @@ public function decrypt_message( $message, $nonce ) {
 	/**
 	 * Gets a collection of bookmarks for this Node
 	 *
+	 * @param  string $url The URL of the Node.
 	 * @return array
 	 */
-	public function get_bookmarks() {
+	public static function get_bookmarks( $url ) {
```

Comment:
This is kind of a nit, but can I ask that instead of making this method static, and then having to change its usage, can we create another static method that could be used in the new places

Then this method could be a shortcode for the other static method. It would just call it passing the url.

The new static method could live in this class, or maybe it makes sense for it to live in the Nodes (plural) class

Reply:
Sure thing! Done in https://github.com/Automattic/newspack-network/pull/56/commits/9ac97753700b7d8298682b8b37a6d551c9e1e49c